taxonomies/ingredients.txt

```diff
@@ -33392,24 +33936,31 @@ fr:pur jus de pommes pressées
 
 <en:apple juice
 en:concentrated apple juice
+da:Æblejuice (koncentrat)
 de:Apfeldicksaft, Apfelsaftkonzentrat
 es:jarabe de manzana
 fi:omenamehutiiviste
 fr:concentré de jus de pomme, jus de pommes concentré
 it:concentrato di succo di mele
 nl:appeldiksap, appelsap ingedikt tot stroop, geconcentreerd appelsap
+pl:Sok jabłkowy (z koncentratu)
+ru:Концентрированный яблочный сок
+sv:äppeljuicekoncentrat
 # openfoodfacts:https://world.openfoodfacts.org/ingredient/concentrated-apple-juice
 # 423 products in 5 languages @2018-10-04
 
 <en:concentrated apple juice
 en:apple juice from concentrate
+da:Æblejuice (koncentrat)
```

Comment:
This is identical to the translation for the concentrate. Maybe `da:Æblejuice fra koncentrat`